cmd/netmesh/main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	cmd := &cobra.Command{Use: "netmesh"}
 	command.SetRootCmd(cmd)
 	nsm := nsmcommand.NewPlugin()
-	interupt := interupthandler.Wrap(nsm)
-	interupt.Init()
-	interupt.Wait()
+	interrupt := interupthandler.Wrap(nsm)
```

Comment:
since you are cleaning up things, maybe you could rename the package name as well s/interupthandler/interrupthandler/

Reply:
good catch